dashboard/src/features/projects/custom-domains/settings/components/HasuraDomain/HasuraDomain.tsx

```diff
@@ -0,0 +1,155 @@
+      const { networking } = data?.config?.hasura?.resources || {};
+      const fqdn = networking?.ingresses?.[0]?.fqdn?.[0];
+      form.reset({ hasura_fqdn: fqdn });
+    }
+  }, [data, loading, form]);
+
+  if (loading) {
+    return (
+      <ActivityIndicator
+        delay={0}
+        label="Loading Hasura Domain..."
+        className="justify-center"
+      />
+    );
+  }
+
+  if (error) {
+    throw error;
+  }
+
+  const { formState, register, watch } = form;
+  const isDirty = Object.keys(formState.dirtyFields).length > 0;
+
+  const hasura_fqdn = watch('hasura_fqdn');
+
+  async function handleSubmit(formValues: HasuraDomainFormValues) {
+    const updateConfigPromise = updateConfig({
+      variables: {
+        appId: currentProject.id,
+        config: {
+          hasura: {
+            resources: {
+              networking: {
+                ingresses: [
+                  {
+                    fqdn: [formValues.hasura_fqdn],
+                  },
+                ],
+              },
+            },
+          },
+        },
+      },
+    });
+
+    try {
+      await toast.promise(
+        updateConfigPromise,
+        {
+          loading: `Hasura domain is being updated...`,
+          success: `Hasura domain has been updated successfully.`,
+          error: getServerError(
+            `An error occurred while trying to update the Hasura domain.`,
+          ),
+        },
+        getToastStyleProps(),
+      );
+
+      form.reset(formValues);
+      await refetchWorkspaceAndProject();
+    } catch {
+      // Note: The toast will handle the error.
+    }
+  }
+
+  return (
+    <FormProvider {...form}>
+      <Form onSubmit={handleSubmit}>
+        <SettingsContainer
+          title="Hasura Domain"
+          description="It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout."
+          docsLink="https://docs.nhost.io/"
+          slotProps={{
+            submitButton: {
+              disabled: !isDirty || maintenanceActive || !isVerified,
+              loading: formState.isSubmitting,
+            },
+            footer: {
+              className: 'hidden',
+            },
+          }}
+          className="grid grid-flow-row gap-y-4 gap-x-4 px-4 lg:grid-cols-5"
+        >
+          <Input
+            {...register('hasura_fqdn')}
+            id="hasura_fqdn"
+            name="hasura_fqdn"
+            type="string"
+            fullWidth
+            className="lg:col-span-2"
+            placeholder="auth.mydomain.dev"
+            error={Boolean(formState.errors.hasura_fqdn?.message)}
+            helperText={formState.errors.hasura_fqdn?.message}
+            slotProps={{ inputRoot: { min: 1, max: 100 } }}
+          />
+          <div className="col-span-5 row-start-2">
+            <VerifyDomain
+              recordType="CNAME"
+              hostname={hasura_fqdn}
+              value={`lb.${currentProject.region.domain}.`}
```

Comment:
This is still wrong, you are missing the region
```suggestion
              value={`lb.${currentProject.region.awsName}.${currentProject.region.domain}.`}
```